### ISSUE TITLE
Unable to get values

### ISSUE THREAD

**Issue by ezh** [jakwings/node-ternary-search-trie#4]
Hello, something strange with searchWithPrefix.
I trying to debug https://github.com/tomi/vscode-rf-language-server/
Autocomplete isn't working. I found that node-ternary-search-trie returns array of undefined values.
For example there is a tree with 3 elements:
```
> this.tree.traverse((a,b) => {console.log(a + " " + b)})
navigate to frontpage [object Object]
navigate to location [object Object]
open default browser [object Object]
TernarySearchTrie {root_: {…}, size_: 3}
```
Values are ok.

But if I'm using searchWithPrefix, I get undefined values:
```
> this.tree.searchWithPrefix("n", (a,b) => {console.log(a + " " + b)})
navigate to frontpage undefined
navigate to location undefined
TernarySearchTrie {root_: {…}, size_: 3}
```
Any ideas what could be wrong?

**Comment by ezh** [comment]
I see that .v is null for the most of elements of the tree.

**Comment by ezh** [comment]
It looks like TernarySearchTrie.prototype.traverse_ already returns value with `r.value.value`
```
      while (!(r = t.next()).done) {
        callback(r.value.key, r.value.value);
      }
```
```
{key: "avigate to frontpage", value: Array(1)}
key: "avigate to frontpage"
value: [UserKeyword]
__proto__: Object
```
And after that TernarySearchTrie.prototype.searchWithPrefix is trying to get `.v`:
```
      this.traverse_(root.m, function(key, node) {
        return callback(prefix + key, node.v);
      });
```
It looks like there should be `return callback(prefix + key, node);` Am I correct?

**Comment by ezh** [comment]
https://github.com/jakwings/node-ternary-search-trie/blob/master/src/trie.coffee#L298

**Comment by iology** [comment]
Yes, you're right. I've fixed it in v5.4.4 :)

**Comment by ezh** [comment]
It looks like there is one more bug -
https://github.com/tomi/node-ternary-search-trie/commit/89978325d2aef63b6b3c5d212d1fff82228fc597

**Comment by ezh** [comment]
Could you check please?

**Comment by iology** [comment]
Thank you. :P fixed in v5.4.5
